### ISSUE TITLE
Integrating hitechcoding/strict-form-mapper-bundle?

### ISSUE THREAD

**Issue by xabbuh** [sensiolabs-de/rich-model-forms-bundle#80]
@zmitic created [a nice bundle](https://github.com/hitechcoding/strict-form-mapper-bundle) which comes with some interesting features. We could join forces and see if there are features we could implement here too.

**Comment by vladyslavstartsev** [comment]
@xabbuh is there a list of things that should be done?

**Comment by xabbuh** [comment]
Not really, sorry. We would need to collect a list of features of hitechcoding/strict-form-mapper-bundle first and then consider which of them could be implemented here.

**Comment by negee7** [comment]
<pif
